lib/medium-admin.php

```diff
@@ -1054,6 +1054,10 @@ private static function _prepare_content($post) {
       $post_content = do_shortcode(wpautop($post->post_content));
     }
 
+		$post_content = apply_filters( 'medium_post_content', $post_content, $post );
+
+		$post_content = do_shortcode(wpautop($post->post_content));
```

Comment:
it seems like this will apply do_shortcode and wpautop twice, after either lines 1050 or 1054?